modules/node_modules/@ncigdc/components/PortalContainer.js

```diff
@@ -27,12 +28,26 @@ import ProgressContainer from '@ncigdc/components/ProgressContainer';
 import ModalContainer from '@ncigdc/components/ModalContainer';
 import NotFound from '@ncigdc/components/NotFound';
 
+import withRouter from '@ncigdc/utils/withRouter';
 import { GlobalTooltip } from '@ncigdc/uikit/Tooltip';
 
 import styled from '@ncigdc/theme/styled';
 
 const SkipLink = styled.a({ position: 'absolute', left: '-999px', backgroundColor: 'black', zIndex: 1000 });
 
+const enhance = compose(
+  connect(store => ({ notifications: store.bannerNotification })),
+  withRouter,
+  lifecycle({
+    componentDidMount(): void {
+      // eslint-disable-next-line fp/no-this, fp/no-mutation
+      this.removeListen = this.props.listen(() => window.scrollTo(0, 0));
```

Comment:
whenever any route change happens. I didn't think about query parameters. I'll fix that :)